docs/source/software/systemwide/r.md

```diff
@@ -1,136 +1,34 @@
 (using-r)=
 # Using R
 
-[R](https://www.r-project.org/) is available as a {ref}`modules <using-module>` and it is also an interactive app on {ref}`Open OnDemand <using-ood>` (OOD). You can also use R with Anaconda.
+[R](https://www.r-project.org/) is available as a {ref}`modules <using-module>` and it is also an interactive app on {ref}`Open OnDemand <using-ood>` (OOD). You can also install R with anaconda or miniconda in a conda environment.
 
-:::{seealso}
-Our {ref}`conda` page and [official Anaconda documentation](https://docs.anaconda.com/anaconda/packages/r-language-pkg-docs/).
-:::
 ## Using R on Open OnDemand
 
-The Open OnDemand application offers several different versions of R accessed through an interactive session with RStudio server in a [Rocker Container](https://rocker-project.org/images/versioned/rstudio). Each version of R is available as a different *flavor* whereby different packages are pre-installed. We host three flavors whose package-lists build on one another in the following order: RStudio < Tidyverse < Geospatial.
+The Open OnDemand application offers several different versions of R accessed through an interactive session with RStudio server in a [Rocker Container](https://rocker-project.org/images/versioned/rstudio). We provide the geospatial *flavor* of Rocker which includes many useful base packages.
 
-In addition to different R packages you will find additional package requirements (e.g, compilers) also present in the three flavors in increasing order.
+In addition to different R packages you will find additional package requirements (e.g, compilers) also present in geospatial flavor.
 
+## Running the Rocker Container in an sbatch script
 
-:::{important}
+If you have installed packages in the Open OnDemand Rstudio application, you may want to use those same packages in an sbatch script (non-interactive).
```

Comment:
To change 'Rstudio' to 'RStudio' with an uppercase 'S' to follow standard naming.

Should we add examples of installing packages inside RStudio application?